templates/cloudformation/apps/aem/full-set/monitoring.yaml

```diff
@@ -15,37 +15,429 @@ Resources:
+              }
+            },
+            {
+              "type": "metric",
+              "x": 0,
+              "y": 27,
+              "width": 12,
+              "height": 3,
+              "properties": {
+                "metrics": [
+                  [ "AWS/ELB", "HTTPCode_Backend_5XX", "LoadBalancerName", "${authorELB}" ],
+                  [ "AWS/ELB", "HTTPCode_Backend_5XX", "LoadBalancerName", "${authorDispatcherELB}" ],
+                  [ "AWS/ELB", "HTTPCode_Backend_5XX", "LoadBalancerName", "${publishDispatcherELB}" ]
+                ],
+                "view": "singleValue",
+                "stacked": false,
+                "region": "ap-southeast-2",
+                "title": "ELB HTTPCode_Backend_5XX Metrics",
+                "period": 300
+              }
+            },
+            {
+              "type": "metric",
+              "x": 12,
+              "y": 27,
+              "width": 12,
+              "height": 3,
+              "properties": {
+                "metrics": [
+                  [ "AWS/ELB", "HTTPCode_Backend_5XX", "LoadBalancerName", "${authorELB}" ],
+                  [ "AWS/ELB", "HTTPCode_Backend_5XX", "LoadBalancerName", "${authorDispatcherELB}" ],
+                  [ "AWS/ELB", "HTTPCode_Backend_5XX", "LoadBalancerName", "${publishDispatcherELB}" ]
+                ],
+                "view": "timeSeries",
+                "stacked": false,
+                "region": "ap-southeast-2",
+                "title": "ELB HTTPCode_Backend_5XX Metrics - Graph",
+                "period": 300
+              }
+            },
+            {
+              "type": "metric",
+              "x": 0,
+              "y": 30,
+              "width": 12,
+              "height": 3,
+              "properties": {
+                "metrics": [
+                  [ "AWS/ELB", "RequestCount", "LoadBalancerName", "${authorELB}" ],
+                  [ "AWS/ELB", "RequestCount", "LoadBalancerName", "${authorDispatcherELB}" ],
+                  [ "AWS/ELB", "RequestCount", "LoadBalancerName", "${publishDispatcherELB}" ]
+                ],
+                "view": "singleValue",
+                "stacked": false,
+                "region": "ap-southeast-2",
+                "title": "ELB RequestCount Metrics",
+                "period": 300
+              }
+            },
+            {
+              "type": "metric",
+              "x": 12,
+              "y": 30,
+              "width": 12,
+              "height": 3,
+              "properties": {
+                "metrics": [
+                  [ "AWS/ELB", "RequestCount", "LoadBalancerName", "${authorELB}" ],
+                  [ "AWS/ELB", "RequestCount", "LoadBalancerName", "${authorDispatcherELB}" ],
+                  [ "AWS/ELB", "RequestCount", "LoadBalancerName", "${publishDispatcherELB}" ]
+                ],
+                "view": "timeSeries",
+                "stacked": false,
+                "region": "ap-southeast-2",
```

Comment:
Region should come from parameter/variable.

Reply:
Replaced region value with variable ${awsRegion}